src/ActionModal/components/DelegationModal.vue

```diff
@@ -385,6 +385,9 @@ export default {
           address: this.address,
           denom: this.denom
         }
+      },
+      update(data) {
+        return data.balance || { amount: 0 }
```

Comment:
shouldn't the API always return this, instead of adding conditionals to all our queries?